patches/minecraft/net/minecraft/network/NetHandlerPlayServer.java.patch

```diff
@@ -37,7 +37,31 @@
              {
                  this.field_147369_b.field_71134_c.func_187251_a(this.field_147369_b, worldserver, itemstack, enumhand, blockpos, enumfacing, p_184337_1_.func_187026_d(), p_184337_1_.func_187025_e(), p_184337_1_.func_187020_f());
              }
-@@ -933,7 +940,9 @@
+@@ -789,12 +796,13 @@
+                 {
+                     this.field_147369_b.func_70634_a(entity.field_70165_t, entity.field_70163_u, entity.field_70161_v);
+                 }
+-                else
++                else if (net.minecraftforge.common.ForgeHooks.onTravelToDimension(this.field_147369_b, entity.field_71093_bK))
+                 {
++                    int prevDimension = this.field_147369_b.field_71093_bK;
+                     WorldServer worldserver1 = this.field_147369_b.func_71121_q();
+                     WorldServer worldserver2 = (WorldServer)entity.field_70170_p;
+                     this.field_147369_b.field_71093_bK = entity.field_71093_bK;
+-                    this.func_147359_a(new SPacketRespawn(this.field_147369_b.field_71093_bK, worldserver1.func_175659_aa(), worldserver1.func_72912_H().func_76067_t(), this.field_147369_b.field_71134_c.func_73081_b()));
++                    this.func_147359_a(new SPacketRespawn(this.field_147369_b.field_71093_bK, worldserver2.func_175659_aa(), worldserver2.func_72912_H().func_76067_t(), this.field_147369_b.field_71134_c.func_73081_b()));
```

Comment:
Is this a bugfix? Or something else?

Reply:
A bugfix, see my original post:
> (as well as the fix from b417594)

Reply:
Even though the other commit sets a bad example, this needs a forge comment with a vanilla bug ID or barring that an explanation of what it does.

Reply:
Yes, the original commit actually had a comment on that patch, sorry. It's been added to both patches now.